### DISCUSSION TITLE
QTable Selection behaving weirdly

### DISCUSSION THREAD

**Discussion by Lalrinfela** [quasarframework/quasar#12684] [General - Components / Directives / etc]
Hi, I have a Tab panels. In each tab panels I have a QTable. When i switch the tab (also fetch data from server), the selection checkbox is pushing other columns to their max width and when i refresh the page, the table display fine again. How to fix this anyone please? I could not find anything that could relate in documentation.
<img width="984" alt="Screenshot 2022-03-07 at 1 19 36 PM" src="https://user-images.githubusercontent.com/1894559/156989616-5a0a83c1-bdd3-448d-a897-f664eb2e4248.png">
<img width="982" alt="Screenshot 2022-03-07 at 1 20 31 PM" src="https://user-images.githubusercontent.com/1894559/156989708-015fdd36-cf75-4241-9074-85b7ae72514f.png">

**Comment by MilosPaunovic** [comment]
This is too specific case. Please create a Codepen reproduction for someone to be able to play with it. You can simulate fetching server data with `setTimeout`.
